### DISCUSSION TITLE
Can't launch storybook in Vue3 project

### DISCUSSION THREAD

**Discussion by randomailer** [storybookjs/storybook#16339] [Help]
can't launch storybook after installation

I have a project with Vue3. I installed all dependencies but it doesn't work.

```
:~/projects/dione/packages/ui$ yarn run storybook:build
yarn run v1.22.10
$ vue-cli-service storybook:build -c .storybook
info @storybook/vue3 v6.3.10
info 
info => Cleaning outputDir: /home/v.titov/projects/dione/packages/ui/storybook-static
info => Loading presets
info => Compiling manager..
info => Compiling preview..
info => Loading 8 other files in "/home/v.titov/projects/dione/packages/ui/.storybook"
info => Adding stories defined in "/home/v.titov/projects/dione/packages/ui/.storybook/main.js"
ERR! Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './dist/vue.esm-browser.js' is not defined by "exports" in /home/v.titov/projects/dione/node_modules/vue/package.json
```

**Comment by shilman** [comment]
Do you a have a reproduction repo you can share? If not, can you create one? See [how to create a repro](https://storybook.js.org/docs/react/contribute/how-to-reproduce). I could not reproduce it on a fresh vue3 project.

**Reply by randomailer** [reply]
Ok thanks, 

I did everything by instruction. https://github.com/randomailer/storybook-test
By the way, I'm using "yarn" for launching.

**Reply by shilman** [reply]
@randomailer I just opened your project https://github.com/randomailer/storybook-test in a Github codespace and it appears to be running fine for me.

**Reply by randomailer** [reply]
Did you use yarn? Which version?
